src/main/webapp/WEB-INF/tags/instructor/course/courseEditAddInstructorPanel.tag

```diff
@@ -0,0 +1,285 @@
+                                    <div class="panel-heading">
+                                        <strong>In general, this instructor can</strong>
+                                    </div>
+                                    
+                                    <div class="panel-body">
+                                        <c:forEach items="${addInstructorPanel.permissionInputGroup1}" var="permissionCheckbox">
+                                            <div class="col-sm-3">
+                                                <input  <c:forEach items="${permissionCheckbox.attributes}" var="attribute">
+                                                            ${attribute.key}="${attribute.value}"
+                                                        </c:forEach> 
+                                                > ${permissionCheckbox.content}
+                                            </div>
+                                        </c:forEach>
+                                        <br>
+                                        <br>
+                                        
+                                        <div class="col-sm-6 border-right-gray">
+                                            <c:forEach items="${addInstructorPanel.permissionInputGroup2}" var="permissionCheckbox">
+                                                <input  <c:forEach items="${permissionCheckbox.attributes}" var="attribute">
+                                                            ${attribute.key}="${attribute.value}"
+                                                        </c:forEach> 
+                                                > ${permissionCheckbox.content}
+                                                <br>
+                                            </c:forEach>
+                                        </div>
+                                        
+                                        <div class="col-sm-5 col-sm-offset-1">
+                                            <c:forEach items="${addInstructorPanel.permissionInputGroup3}" var="permissionCheckbox">
+                                                <input  <c:forEach items="${permissionCheckbox.attributes}" var="attribute">
+                                                            ${attribute.key}="${attribute.value}"
+                                                        </c:forEach> 
+                                                > ${permissionCheckbox.content}
+                                                <br>
+                                            </c:forEach>
+                                        </div>
+                                    </div>
+                                </div>
+                                
+                                <c:if test="${not empty addInstructorPanel.sectionRows}">
+                                    <c:forEach items="${addInstructorPanel.sectionRows}" var="sectionRow" varStatus="i">
```

Comment:
Maybe put the `<c:if>` after the `<c:forEach>` like for the `feedbackSessions` below?
This will lower the indentation of this whole block while achieving the same effect.